client/src/pages/Jessica.jsx

```diff
@@ -1,13 +1,57 @@
 // @flow
 import React from 'react';
 
-import '../css/Home.css';
+import '../css/Jessica.css';
+import bg from '../utils/images/bg.jpg';
```

Comment:
in general it's best to avoid abbreviations, makes the component easier to understand at a glance.